web/src/features/panels/ranking-panel/InfoText.tsx

```diff
@@ -46,10 +59,7 @@ export default function InfoText() {
         .
       </p>
       <p>
-        {__('footer.faq-text')}{' '}
-        <Link to={createToWithState('/faq')}>
-          <span data-test-id="faq-link">{__('footer.faq')}</span>
-        </Link>
+        {__('footer.faq-text')} <FAQLink>{__('footer.faq')}</FAQLink>
```

Comment:
Would it not make sense to remove this link altogether?

Reply:
It could, but I figured it was worth keeping an extra reference for now :)